app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -156,8 +156,12 @@
     private boolean showRelatedStreams;
     private boolean showDescription;
     private String selectedTabTag;
-    @AttrRes @NonNull final List<Integer> tabIcons = new ArrayList<>();
-    @StringRes @NonNull final List<Integer> tabContentDescriptions = new ArrayList<>();
+    @AttrRes
+    @NonNull
+    final List<Integer> tabIcons = new ArrayList<>();
+    @StringRes
+    @NonNull
+    final List<Integer> tabContentDescriptions = new ArrayList<>();
```

Comment:
Restore this

Reply:
My IDE automatically reformat the code , I had to do editing in github file itself :)
Is there some way so that it does not do this ?

Reply:
What IDE are you using? Android studio doesn't do that usually

Reply:
Android Studio with some altered settings